terraform/modules/prom-ec2/paas-config/prometheus.conf.tpl

```diff
@@ -4,26 +4,67 @@ global:
 alerting:
   alertmanagers:
   - scheme: http
-    static_configs:
-      - targets: [${alertmanager_dns_names}]
+    ec2_sd_configs:
+      - region: eu-west-1
```

Comment:
Optional: Maybe we could be passing the region in as a var?

Reply:
YAGNI :)